app/src/main/java/com/sapuseven/untis/preferences/DataStorePreferences.kt

```diff
@@ -234,6 +235,11 @@ val BaseComposeActivity.dataStorePreferences: DataStorePreferences
 				"preference_school_background",
 				defaultValue = emptySet()
 			),
+			absencesOrder = this.stringDataStore(
+				currentUserId(),
+				"info_center_absences_order",
+				defaultValue = "{\"type\":\"com.sapuseven.untis.ui.activities.AbsenceOrder.CurrentSchoolYear\",\"orderType\":{\"type\":\"com.sapuseven.untis.ui.activities.OrderType.Descending\",\"showOnlyUnexcused\":false}}"
+			)
```

Comment:
I think it would be better to use separate data stores for each filter preference.
That way you don't need to worry about (de)serialization causing problems.
For example, adding or changing preferences will be easier if handled separately.